spec/services/invite_schools_spec.rb

```diff
@@ -73,4 +73,56 @@
       end
     end
   end
+
+  describe "#sent_email_recently?" do
+    it "is false when the school has not been emailed" do
+      expect(invite_schools.sent_email_recently?(school)).to eq false
+    end
+
+    context "when the school has been emailed more than 24 hours ago" do
```

Comment:
So that's an interesting question. Do we want to make it 24 hours? Would it be annoying if someone tries sending it in the afternoon on Monday, and in the morning on Tuesday? Something to maybe mention to product people, I wouldn't change it here.